packages/core/src/core.ts

```diff
@@ -204,3 +214,20 @@ export function saveState(name: string, value: string): void {
 export function getState(name: string): string {
   return process.env[`STATE_${name}`] || ''
 }
+
+/**
+ * Sanatizes an input into a string so it can be passed into issueCommand safely
+ * @param input input to sanitize into a string
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function sanitizeInput(input: any): string {
+  if (input == null) {
+    return ''
+  } else if (typeof input === 'string' || input instanceof String) {
+    return input as string
+  } else if (input instanceof Error) {
```

Comment:
is converting error to string only something the error/setFailed functions should do? I'm thinking about the description for the setOutput function... seems like weird behavior.